README.md

```diff
@@ -11,4 +11,57 @@
 
 .Net Client for RedisTimeSeries
 
+
+## API
+The complete documentation of RedisTimeSeries's commands can be found at [RedisTimeSeries's website](http://redistimeseries.io/).
+
+## Usage example
```

Comment:
I think only the simple example should be in the readme. Have a link to the advanced API example files